<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataLoaderPanel</class>
 <widget class="QWidget" name="dataLoaderPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>72</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Select File to Load</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_clearData">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Clear Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_loadData">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Load Data</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit_currentFile"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
